stargate-starter/src/main/java/io/stargate/starter/Starter.java

```diff
@@ -373,6 +392,28 @@ public void start() throws BundleException {
       Bundle b = context.installBundle(jar.toURI().toString());
       bundleList.add(b);
     }
+
+    if (nodetool) {
+      for (Bundle bundle : bundleList) {
+        try {
+          Class<?> tool = bundle.loadClass("org.apache.cassandra.tools.NodeTool");
```

Comment:
Do we just try to run it on every bundle until we find it? Would it be possible to short circuit and go direct to the bundle rather than looping?

Reply:
yes. I had that initially, but for some reason went back to the loop. I'll update to use the persistence bundle directly.

Reply:
Oh I see down below where you put it as the first one